guides/source/configuring.md

```diff
@@ -305,7 +305,9 @@ All these configuration options are delegated to the `I18n` library.
   `:all` which always dumps all schemas regardless of the schema_search_path,
   or a string of comma separated schemas.
 
-* `config.active_record.belongs_to_required_by_default` is a boolean value and controls whether `belongs_to` association is required by default.
+* `config.active_record.belongs_to_required_by_default` is a boolean
+  value and controls whether validation error is triggered by default if
```

Comment:
I would replace `validation error is triggered by default` to `a record fails validation`